CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
+## 0.9.2 (?)
+- informational - The `redHatDependencyAnalyticsReportFilePath` setting name has changed to `reportFilePath`. If you had a custom file path set for `redHatDependencyAnalyticsReportFilePath`, then you need to add your custom file path to the `reportFilePath` setting.
+- enhancement - Added a vulnerability severity alert level setting for the user to receive inline notifications for just errors or warnings. See [PR#674](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/674) for details.
+- fixes - Fixed an issue with the `codeActionsMap` call. When multiple manifest documents are open that have the same dependency, one of the document entries gets deleted. This gave a wrong result in the analysis. See [PR#236](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/236) for details.
+- fixes - Fixed an issue in the Exhort Javascript API. This fix enables and supports analysis of `pom.xml` manifests that include local modules, and a parent Project Object Model (POM). See the [PR#237](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/237) for details.
+- fixes - Fixed an issue with the analysis report not displaying because of spaces in the manifest file path on Windows. See [PR#100](https://github.com/RHEcosystemAppEng/exhort-javascript-api/pull/100) for details.
```

Comment:
This is not exclusively a Windows-related problem, even though it was identified primarily in Windows-type operating systems at the time of this fix. The reason for this is that Windows paths often contain spaces in their absolute paths. While this bug can be replicated in any operating system, it is generally uncommon to have directory names with spaces in paths, hence the low probability of its detection.
Please consider if mentioning 'Windows' in this case is necessary.

Reply:
Okay, you mentioned Windows in your post, and the related issues you linked all mentioned Windows.  Hence, that's why I mentioned Windows.  I'll go ahead and remove Windows.

Reply:
I can see why that was confusing, sorry for not expanding of this before hand. Thank you for the fix :)